Xamarin.Forms.Platform.iOS/EventTracker.cs

```diff
@@ -133,22 +138,81 @@ protected virtual NativeGestureRecognizer GetNativeRecognizer(IGestureRecognizer
 				return uiRecognizer;
 			}
 #else
-            if (tapRecognizer != null)
-            {
-                var returnAction = new Action(() =>
-                {
-                    var tapGestureRecognizer = weakRecognizer.Target as TapGestureRecognizer;
-                    var eventTracker = weakEventTracker.Target as EventTracker;
-                    var view = eventTracker?._renderer?.Element as View;
-
-                    if (tapGestureRecognizer != null && view != null)
-                        tapGestureRecognizer.SendTapped(view);
-                });
-                var uiRecognizer = CreateTapRecognizer(tapRecognizer.NumberOfTapsRequired, returnAction);
-                return uiRecognizer;
-            }                
+			if (tapRecognizer != null)
+			{
+				var returnAction = new Action<UITapGestureRecognizer>((sender) =>
+				{
+					var eventTracker = weakEventTracker.Target as EventTracker;
+					var view = eventTracker?._renderer?.Element as View;
+
+					var originPoint = sender.LocationInView(null);
+					originPoint = UIApplication.SharedApplication.KeyWindow.ConvertPointToView(originPoint, eventTracker._renderer.NativeView);
+
+					var childGestures = view.GetChildElements(new Point(originPoint.X, originPoint.Y));
+
+					if (childGestures?.GetChildGesturesFor<TapGestureRecognizer>(x => x.NumberOfTapsRequired == (int)sender.NumberOfTapsRequired).Count() > 0)
+						return;
+
+					if (weakRecognizer.Target is TapGestureRecognizer tapGestureRecognizer && view != null)
+						tapGestureRecognizer.SendTapped(view);
+
+				});
+
+				var uiRecognizer = CreateTapRecognizer(tapRecognizer.NumberOfTapsRequired, returnAction);
+				return uiRecognizer;
+			}
 #endif
 
+			if (recognizer is ChildGestureRecognizer childRecognizer)
```

Comment:
This is where I use the ChildGestureRecognizer to tell if its a child element. This allows me to use different logic in the returnAction, as compared to the regular TapGestureRecognizers. They have the inverse logic to each other. e.g. If the ChildGestureRecognizer finds something in the ChildElements GestureRecognizers it acts upon them. If the regular TapGestureRecognizers see that a child element will recognize it, it ignores the handling of it.